### PR TITLE
Avatar Component

### PR DESCRIPTION
**Pull request by kaustubh-jsr**
Added the avatar component

### CODE REVIEW

File: components/avatar.css [hunk 1]
```diff
@@ -0,0 +1,54 @@
+.avatar__example-demo-container{
+    display: flex;
+    align-items: flex-end;
+    justify-content: space-evenly;
+    flex-wrap: wrap;
+}
+
+.img-round{
+    border-radius: 50%;
```
Comment (Naman-Saxena1) [inline components/avatar.css:2]: You can use object-fit: cover property, then your avatar pics will look better.

File: documentation.html [hunk 2]
```diff
@@ -151,74 +151,69 @@
...
-                    A simple secondary alert—check it out!
+                <div class="component__example-demo avatar__example-demo-container">
+                    <div class="avatar avatar-xl">
+                        <img class="img-responsive img-round" loading="lazy" src="./resources/avatar-img-1.jpg" alt="avatar" srcset="">
                     </div>
-                    <div class="alert alert-success">
-                        <span><i class="fas fa-check-circle"></i></span> 
-                        A simple success alert—check it out!
+                    <div class="avatar avatar-lg">
+                        <img class="img-responsive img-round" loading="lazy" src="./resources/avatar-img-2.jpg" alt="avatar" srcset="">
```
Comment (Naman-Saxena1) [inline documentation.html]: If you are not using srcset property, you can remove it.

**Review by Naman-Saxena1 — CHANGES_REQUESTED** [review]

File: components/avatar.css [hunk 3]
```diff
@@ -0,0 +1,54 @@
+.avatar__example-demo-container{
```
Comment (anasm11) [inline components/avatar.css]: This class can be given in css file of documentation as it is project specific.

**Review by anasm11 — COMMENTED** [review]
Suggested a change. Rest all looks good.

File: documentation.html (hunk 2 above)
Comment (kaustubh-jsr) [reply documentation.html]: Done

File: components/avatar.css (hunk 1 above)
Comment (kaustubh-jsr) [reply components/avatar.css:2]: Thanks for this suggestion. This really helped.I have pushed the changes.

File: components/avatar.css (hunk 3 above)
Comment (kaustubh-jsr) [reply components/avatar.css]: Nice catch there. Thanks for that. Refactored and pushed.

File: components/avatar.css (hunk 1 above)
Comment (Ankur9669) [reply components/avatar.css:2]: Use object-fit cover in img responsive class. It would be great there

File: components/avatar.css [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+    border-radius: 50%;
+}
+
+.avatar-text{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: #1D53FF;
+    color: white;
+    font-weight: bold;
```
Comment (Ankur9669) [inline components/avatar.css:21]: You can try to use css variables here

**Review by Ankur9669 — APPROVED** [review]
The code looks awesome buddy😀

File: components/avatar.css (hunk 4 above)
Comment (kaustubh-jsr) [reply components/avatar.css:21]: Good catch there. Refactored and pushed. Thanks Ankur.

Comment (kaustubh-jsr) [reply components/avatar.css:21]: Good catch there. Refactored ,thanks Ankur.

**Pull request merged**